lib/wings/hydra/works/services/add_file_node_to_file_set.rb

```diff
@@ -45,14 +45,15 @@ def update
 
         # @param [RDF::URI] the identified use of the file (e.g. Valkyrie::Vocab::PCDMUse.OriginalFile, Valkyrie::Vocab::PCDMUse.ThumbnailImage, etc.)
         # @param [true, false] update_existing when true, try to retrieve existing element before building one
-        def find_or_create_file(use, update_existing)
```

Comment:
This code grabbed from Hydra::Works::AddFileToFileSet #find_or_create_file_for_rdf_uri does not work.  It fails to create the original_file association which is needed in other places in Hyrax (e.g. CharacterizeJob).

So using the code from #find_or_create_file_for_symbol.  Requires that the URI passed in be converted to a symbol first.